### PR TITLE
Deploy yum SVCs on build01

### PR DESCRIPTION
**Pull request by hongkailiu**
Fixing https://github.com/openshift/release/pull/6331

https://coreos.slack.com/archives/GB7NB0CUC/p1576086394491000?thread_ts=1576009509.458800&cid=GB7NB0CUC

/cc @openshift/openshift-team-developer-productivity-test-platform 

/hold

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
We should think about how we want to ensure that there is never any drift. Maybe add a presubmit that runs `diff` on these two dirs and ensures they are identical

**Comment by hongkailiu** [comment]
> We should think about how we want to ensure that there is never any drift. Maybe add a presubmit that runs `diff` on these two dirs and ensures they are identical

Done.

Rehearsal passed.
https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/6354/rehearse-6354-pull-ci-openshift-release-master-build-farm-consistency/9

```bash
$ oc logs  -n ci-op-rr2w1g31 build-farm-consistency
copying ns-ttl-controller assets ...
cp -fv ./core-services/ci-ns-ttl-controller/ci-ns-ttl-controller_dc.yaml ./clusters/build-clusters/01_cluster/openshift/ci-ns-ttl-controller/ci-ns-ttl-controller_dc.yaml
'./core-services/ci-ns-ttl-controller/ci-ns-ttl-controller_dc.yaml' -> './clusters/build-clusters/01_cluster/openshift/ci-ns-ttl-controller/ci-ns-ttl-controller_dc.yaml'
copying rpms-ocp assets ...
#ensure the source is there
find ./core-services/release-controller/ -name "rpms-ocp-*.yaml" | egrep '.*'
./core-services/release-controller/rpms-ocp-4.1.yaml
./core-services/release-controller/rpms-ocp-4.2.yaml
./core-services/release-controller/rpms-ocp-4.3.yaml
./core-services/release-controller/rpms-ocp-4.4.yaml
./core-services/release-controller/rpms-ocp-4.5.yaml
find ./core-services/release-controller/ -name "rpms-ocp-*.yaml" -exec cp -v {} ./clusters/build-clusters/01_cluster/openshift/release-controller/ \;
'./core-services/release-controller/rpms-ocp-4.1.yaml' -> './clusters/build-clusters/01_cluster/openshift/release-controller/rpms-ocp-4.1.yaml'
'./core-services/release-controller/rpms-ocp-4.2.yaml' -> './clusters/build-clusters/01_cluster/openshift/release-controller/rpms-ocp-4.2.yaml'
'./core-services/release-controller/rpms-ocp-4.3.yaml' -> './clusters/build-clusters/01_cluster/openshift/release-controller/rpms-ocp-4.3.yaml'
'./core-services/release-controller/rpms-ocp-4.4.yaml' -> './clusters/build-clusters/01_cluster/openshift/release-controller/rpms-ocp-4.4.yaml'
'./core-services/release-controller/rpms-ocp-4.5.yaml' -> './clusters/build-clusters/01_cluster/openshift/release-controller/rpms-ocp-4.5.yaml'
git status -s ./clusters/build-clusters/
test -z "$(git status -s ./clusters/build-clusters/)"

```


/hold cancel

**Comment by hongkailiu** [comment]
Feeling much better with this job def.
https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/6354/rehearse-6354-pull-ci-openshift-release-master-build-farm-consistency/17

File: ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml [hunk 1]
```diff
@@ -18,6 +18,37 @@ presubmits:
...
+      containers:
+      - args:
+        - -c
+        - |
+          set -o errexit
+          set -o nounset
+          set -o pipefail
+          pwd
+          ls -al
+          echo "${GOPATH}/src/github.com/openshift/release/"
```
Comment (stevekuznetsov) [inline ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml]: This is your working directory, why do you need to `-C`?

File: ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml [hunk 2]
```diff
@@ -18,6 +18,37 @@ presubmits:
...
+          make -C "${GOPATH}/src/github.com/openshift/release/" build-farm-consistency
```
Comment (stevekuznetsov) [inline ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml]: Why do we need all this? Couldn't you just run one `diff` command?

File: Makefile [hunk 3]
```diff
@@ -305,6 +305,17 @@ prow-monitoring:
 	make -C cluster/ci/monitoring prow-monitoring-deploy
 .PHONY: prow-monitoring
 
+build-farm-consistency:
```
Comment (stevekuznetsov) [inline Makefile]: I don't think we need this complexity for a `cp`

Comment (hongkailiu) [reply Makefile]: Done.

File: ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml (hunk 1 above)
Comment (hongkailiu) [reply ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml]: Removed.
Debugging when i have `skip_clone: true`

File: Makefile (hunk 3 above)
Comment (hongkailiu) [reply Makefile]: Removed.

File: ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml (hunk 2 above)
Comment (hongkailiu) [reply ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml]: Yes.
Changed.

**Comment by hongkailiu** [comment]
Latest rehearsal
https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/6354/rehearse-6354-pull-ci-openshift-release-master-build-farm-consistency/18

**Comment by hongkailiu** [comment]
Looks good

```bash
$ oc --kubeconfig build01.config get pod -n ocp 
NAME                                  READY   STATUS    RESTARTS   AGE
base-4-1-5cb66cf88d-dndxc             1/1     Running   0          64s
base-4-1-5cb66cf88d-nlhsm             1/1     Running   0          73s
base-4-2-9578bb4ff-nbcgl              1/1     Running   0          73s
base-4-2-9578bb4ff-pq6xh              1/1     Running   0          64s
base-4-2-s390x-5dbcf7f487-4g4nh       1/1     Running   0          63s
base-4-2-s390x-5dbcf7f487-dz2vb       1/1     Running   0          72s
base-4-3-587854b594-mmhr2             1/1     Running   0          23h
base-4-3-587854b594-vpjnv             1/1     Running   0          23h
base-4-3-ppc64le-787b858477-6l8qj     1/1     Running   0          23h
base-4-3-ppc64le-787b858477-s2vq6     1/1     Running   0          23h
base-4-3-rhel8-9f56c79c8-kqtt7        1/1     Running   0          23h
base-4-3-rhel8-9f56c79c8-x82cz        1/1     Running   0          23h
base-4-3-s390x-6c685bfbf4-dvmxn       1/1     Running   0          23h
base-4-3-s390x-6c685bfbf4-rdbbb       1/1     Running   0          23h
base-4-4-9b5485fb5-dmbfk              1/1     Running   0          5h27m
base-4-4-9b5485fb5-t66cs              1/1     Running   0          5h27m
base-4-4-ppc64le-6f6c747fbb-8xn4k     1/1     Running   0          5h27m
base-4-4-ppc64le-6f6c747fbb-x62xx     1/1     Running   0          5h27m
base-4-4-rhel8-68cdb6cd9c-5lpbx       1/1     Running   0          5h27m
base-4-4-rhel8-68cdb6cd9c-5mvkc       1/1     Running   0          5h27m
base-4-4-s390x-58978f7697-drwjf       1/1     Running   0          5h27m
base-4-4-s390x-58978f7697-jf4nq       1/1     Running   0          5h27m
base-4-5-6ff547d64-5g7kg              1/1     Running   0          61s
base-4-5-6ff547d64-z2dtb              1/1     Running   0          69s
base-4-5-ppc64le-67858b9667-28ttz     1/1     Running   0          69s
base-4-5-ppc64le-67858b9667-f9lg4     1/1     Running   0          60s
base-4-5-rhel8-6b6b5ddd4d-5hprr       1/1     Running   0          60s
base-4-5-rhel8-6b6b5ddd4d-9fbxm       1/1     Running   0          69s
base-4-5-s390x-869fd8f7b8-qbmj9       1/1     Running   0          60s
base-4-5-s390x-869fd8f7b8-vshr2       1/1     Running   0          69s
base-openstack-4-2-66fbd758b4-4gj2g   1/1     Running   0          63s
base-openstack-4-2-66fbd758b4-lxvdt   1/1     Running   0          73s
base-openstack-4-3-777894bbf7-bv9m2   1/1     Running   0          23h
base-openstack-4-3-777894bbf7-vghdn   1/1     Running   0          23h
base-openstack-4-4-6d7d68bcdb-8dd9l   1/1     Running   0          5h27m
base-openstack-4-4-6d7d68bcdb-q2zbv   1/1     Running   0          5h27m
base-openstack-4-5-5c9d55c567-2t5s8   1/1     Running   0          60s
base-openstack-4-5-5c9d55c567-ksfmp   1/1     Running   0          69s
openstack-beta-4-3-5c74dbdf6-mwwlg    1/1     Running   0          23h
openstack-beta-4-3-5c74dbdf6-vkwh2    1/1     Running   0          23h
openstack-beta-4-4-b98d64fbf-69zn4    1/1     Running   0          5h27m
openstack-beta-4-4-b98d64fbf-sc6bc    1/1     Running   0          5h27m
openstack-beta-4-5-55fb5cbf8b-48jf5   1/1     Running   0          60s
openstack-beta-4-5-55fb5cbf8b-x8t6c   1/1     Running   0          69s

```

**Pull request merged**